app/assets/stylesheets/govuk_publishing_components/components/_subscription-links.scss

```diff
@@ -8,7 +8,7 @@
 
   .gem-c-subscription-links__list {
     list-style: none;
-    margin: 0 (-govuk-spacing(3) / 2);
+    margin: 0 - (govuk-spacing(3)) / 2;
```

Comment:
I'm not sure this is quite right still but the current version is broken so I'm not sure what it's supposed to be! From this code it looks like the margin top and bottom is supposed to be `0`, with a negative margin left and right, but in the [deployment](https://govuk-publishing-compon-pr-914.herokuapp.com/component-guide/subscription-links) it's appearing with a negative margin all the way round.

Reply:
It's the same with what happened in the previous next component. I saw it broken in live too, I'll dig a bit and see how it used to look like